packages/ui/src/components/Datepicker/Datepicker.tsx

```diff
@@ -82,21 +82,23 @@ export interface DatepickerRef {
   clear: () => void;
 }
 
-export interface DatepickerProps extends Omit<TextInputProps, "theme"> {
+export interface DatepickerProps extends Omit<TextInputProps, "theme" | "onChange"> {
+  defaultDate?: Date;
   open?: boolean;
   inline?: boolean;
   autoHide?: boolean;
   showClearButton?: boolean;
   labelClearButton?: string;
   showTodayButton?: boolean;
   labelTodayButton?: string;
-  defaultDate?: Date;
   minDate?: Date;
   maxDate?: Date;
   language?: string;
   weekStart?: WeekStart;
   theme?: DeepPartial<FlowbiteDatepickerTheme>;
-  onSelectedDateChanged?: (date: Date) => void;
+  onChange?: (date: Date) => void;
```

Comment:
It is a breaking change, right?

Reply:
It has become so.

Reply:
Please, provide some information about it in the PR description.